kotlin/perform-spatial-operations/README.metadata.json

```diff
@@ -3,30 +3,37 @@
     "description": "Perform geometry-on-geometry spatial operations by using GeometryEngine. Two input Polygons are added as Graphics to a GraphicsOverlay and displayed in a MapView.",
     "ignore": false,
     "images": [
-        "spatial-operations.png"
+        "perform-spatial-operations.png"
     ],
     "keywords": [
-        "Edit and Manage Data",
+	"analysis",
+	"combine",
+	"difference",
+	"geometry",
+	"intersection",
+	"merge",
+	"polygon",
+	"union",
         "GeometryEngine.difference",
```

Comment:
this and the following should be removed as they come under the relevant api section rather than keywords (GeometryEngine.difference through to GraphicsOverlay)